be/tool/beck.c

```diff
@@ -631,6 +636,9 @@ int main(int argc, char **argv)
 	if (dry_run)
 		printf("Running in read-only mode.\n");
 
+	if (mmap_be_segment)
```

Comment:
It will be better if we have this check above line 636 dry_run check. That way the line 637 printf get displayed to user.